packages/tslint-config/package.json

```
@@ -0,0 +1,16 @@
+{
+  "description": "Wire TSLint config",
+  "license": "GPL-3.0",
+  "main": "tslint.json",
+  "name": "@wireapp/tslint-config",
+  "peerDependencies": {
+    "prettier": ">=1",
```

```suggestion
    "prettier": ">= 1",
```
same for the other peer dependencies